skin/frontend/base/default/algoliasearch/algoliasearch.css

```diff
@@ -1211,4 +1211,10 @@ a.ais-current-refined-values--link:hover
 
 #algolia-right-container .infos.hidden {
 	display: none;
+}
+
+.algolia-clearfix:after {
+	content:"";
+	display:block;
```

Comment:
Actually in the link he's favoring `block`:
<img width="543" alt="screenshot 2017-02-24 09 55 14" src="https://cloud.githubusercontent.com/assets/1058144/23296918/81154e02-fa77-11e6-8fd5-fd61460f263b.png">